server/app.js

```diff
@@ -127,7 +127,7 @@ module.exports = function createApp({
   // Secure code best practice - see:
   // 1. https://expressjs.com/en/advanced/best-practice-security.html,
   // 2. https://www.npmjs.com/package/helmet
-  app.use(helmet())
+  app.use(helmet({ contentSecurityPolicy: false }))
```

Comment:
https://www.npmjs.com/package/helmet suggests the {contentSecurityPolicy : false} is already captured by helmet()

Reply:
The default is true, the doc tells you how to disable it which is what we're doing here